redata/backends/exasol.py

```diff
@@ -114,6 +114,20 @@ def check_data_volume(self, table, where_timecol):
             fetch_dict=True,
         ).fetchone()
         return SimpleNamespace(**result)
+    
+    def check_data_volume_diff(self, table, where_timecol):
```

Comment:
@jwarlander this is my guess how it should look like on exasol :) So let me know if something doesn't look good. Also if using groupby slows query down, it's definitely possible to implement this without it (by doing separate filter queries by day)

Not sure if that's clear, if not let me know :)